support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -145,6 +145,27 @@ export const tests: Tests = {
 		targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages,
 		excludeCountriesSubjectToContributionsOnlyAmounts: true,
 	},
+	july2024PriceRise: {
+		variants: [
+			{
```

Comment:
If we're intending on using this AB test to launch the changes then we won't need the `control`? We should probably get rid of it from here as if we leave it in and enable the test 50% of users will see the old prices on Tier 3.